src/GafferOFXModule/GafferOFXModule.cpp

```
@@ -0,0 +1,105 @@
+        );
+    }
+};
+
+bool createPluginInstanceWrapper( OFXImageNode& node )
+{
+	IECorePython::ScopedGILRelease gilRelease;
+	return node.createPluginInstance();
+
+}
+
+std::pair<double, double> effectInstanceProjectSizeWrapper( OFXImageNode& node )
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)